.env.defaults

```
@@ -19,7 +19,13 @@ GRIST_TEAM_SITE=anct
+WORKER_BROKER_URL=redis://redis:6379/0
 
-FLOWER_UNAUTHENTICATED_API=true
\ No newline at end of file
+# Worker dashboard. In production (scripts/scalingo_run_web) it is exposed at
+# /$WORKER_DASHBOARD_URL behind Caddy and only started when the IP allowlist and
+# basic-auth credentials below are both set (fail closed):
+#   WORKER_DASHBOARD_URL             URL path it is mounted at (default "worker-dashboard")
+#   WORKER_DASHBOARD_ALLOWED_IPS     space-separated IPs/CIDRs allowed (e.g. "1.2.3.4 5.6.0.0/16")
+#   WORKER_DASHBOARD_BASICAUTH_USER  basic-auth username
+#   WORKER_DASHBOARD_BASICAUTH_HASH  bcrypt hash from `caddy hash-password`
+#   WORKER_DASHBOARD_PORT            internal port Caddy proxies to (default 8090)
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)